content/docs/command-reference/remote/modify.md

```diff
@@ -589,6 +589,9 @@ more information.
     to the remote in the form: `custom_auth_header: password`.
     `custom_auth_header` and `password` (or `ask_password`) parameters should
     also be configured.
+  - `method` - override the
+    [HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)
+    to use for file uploads (e.g. `PUT`/`POST`). By default, `POST` is used.
```

Comment:
I'm not sure I get how this works. Can you guys provide a quick example? What this describes so far is:

```console
$ dvc remote modify  myremote auth method
```

How do you set the actual method? Why is the method a type of auth? Thanks

Reply:
> $ dvc remote modify myremote method PUT

Reply:
OK so this bullet shouldn't be nested under `auth` then 🙂

Reply:
Or are both needed?

Reply:
Oops, didn't notice that. Good catch! Thanks!